config/browse_everything_providers.yml

```diff
@@ -12,8 +12,8 @@ file_system:
 #   :client_id: YOUR_BOX_CLIENT_ID
 #   :client_secret: YOUR_BOX_CLIENT_SECRET
 # google_drive:
-#   :client_id: YOUR_GOOGLE_API_CLIENT_ID
```

Comment:
this is incorrect. Google drive and Google analytics are two different things.

Reply:
reverted!